blocks/hero-block/features/hero/default.jsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+
+// Arc Themes Components - Base set of components used to compose blocks
+// https://github.com/WPMedia/arc-themes-components/
+import {
+	Button,
+	Heading,
+	HeadingSection,
+	Image,
+	Paragraph,
+	Stack,
+} from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-hero";
+
+function Hero({ customFields }) {
+	const {
+		layout = "overlay",
+		alignment = "center",
+		description,
+		headline,
+		imageURL,
+		link1Action,
+		link1Text,
+		link1Type,
+		link2Action,
+		link2Text,
+		link2Type,
+		subHeadline,
+		variant = "dark",
+	} = customFields;
+
+	const classes = [
+		BLOCK_CLASS_NAME,
+		variant === "dark" ? `${BLOCK_CLASS_NAME}--dark` : `${BLOCK_CLASS_NAME}--light`,
+	].join(" ");
+
+	return (
+		<div className={classes}>
+			<Image className="b-hero--image" src={imageURL} alt="" />
+			<Stack
+				data-style-layout={layout}
+				alignment={alignment}
+				className={`${BLOCK_CLASS_NAME}--text ${layout} ${
+					alignment === "left" ? `${BLOCK_CLASS_NAME}--left` : `${BLOCK_CLASS_NAME}--center`
+				}`}
+			>
+				<HeadingSection>
+					{headline ? <Heading className="b-hero--headline">{headline}</Heading> : null}
+					{subHeadline ? (
+						<HeadingSection>
+							<Heading className="b-hero--subheadline">{subHeadline}</Heading>
+						</HeadingSection>
+					) : null}
+					{description ? <Paragraph className="b-hero--paragraph">{description}</Paragraph> : null}
+
+					{(link1Action && link1Text) || (link2Action && link2Text) ? (
+						<Stack
+							className="b-hero--buttonrow"
+							direction="horizontal"
+							justification={alignment === "center" ? "center" : "start"}
+						>
+							<Button href={link1Action} variant={link1Type}>
+								{link1Text}
+							</Button>
+							<Button href={link2Action} variant={link2Type}>
+								{link2Text}
+							</Button>
+						</Stack>
+					) : null}
```

Comment:
not sure if i follow the logic here. I would expect to render each link button only if there's that specific link. For example, if there's link1action and link1text truthy, then link2action and link2text would still be rendered here. can you check if the buttons exist individually as well? 

The top-level logic is ok for rendering the stack container.

Reply:
Let me know if you need any clarification here @visionbegin. This appears to still be in the code

Reply:
I got you.

Reply:
Also see the one button story.